portal/src/main/webapp/index.jsp

```diff
@@ -0,0 +1,76 @@
+<%
```

Comment:
@inodb This is a simple jsp which serves as a replacement for the .ejs file.   It's possible that we can just get rid of this eventually and simply serve the output of the ejs file from the WAR file.    But there may be reasons we still want to allow server to write some stuff to the page.   For example, if we want to load configuration information here instead of forcing the app to fetch it.   We can easily allow the application to do both/either.